python/cudf/cudf/tests/test_rank.py

```diff
@@ -125,9 +125,11 @@ def test_rank_error_arguments(pdf):
     )
 
 
+rng = np.random.default_rng(seed=0)
+
 sort_group_args = [
     np.full((3,), np.nan),
-    100 * np.random.random(10),
+    100 * rng.random(10),
```

Comment:
TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

Reply:
Fixed it in this PR itself.